distribution/docs/src/main/resources/content/_managing/attributes-added-by-pre-authorization-plugins.adoc

```diff
@@ -7,7 +7,7 @@
 
 === {title}
 
-The <<_pre_authorization_plugins,Pre-Authorization Plugins>> provide an opportunity to take action before any security rules are applied.
+The <<{architecture-prefix}pre_authorization_plugins,Pre-Authorization Plugins>> provide an opportunity to take action before any security rules are applied.
 
-* The <<_metacard_ingest_network_plugin,Metacard Ingest Network Plugin>> is a configurable plugin that allows the conditional insertion of new attributes on metacards during ingest based on network information from the ingest request.
-See <<_configuring_the_metacard_ingest_network_plugin,Configuring the Metacard Ingest Network Plugin>> for configuration details.
\ No newline at end of file
+* The <<{architecture-prefix}metacard_ingest_network_plugin,Metacard Ingest Network Plugin>> is a configurable plugin that allows the conditional insertion of new attributes on metacards during ingest based on network information from the ingest request.
+See <<{architecture-prefix}configuring_the{architecture-prefix}metacard_ingest_network_plugin,Configuring the Metacard Ingest Network Plugin>> for configuration details.
```

Comment:
❓ Why does this one have a prefix in the middle of the text?

Reply:
typo. fixed